docs/SUMMARY.md

```
@@ -1,19 +1,27 @@
 # Summary
 
-* [Overview](README.md)
+* [System Overview](README.md)
 
-## 🎓 Tutorials
+## Explanation & Architecture
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)